### PR TITLE
Modified the format as storage,collector,agent,UI&Query services

### PR DESCRIPTION
**Pull request by dexter816**
Changes made other than reordering : added link to source repository for
create.sh script, added example to collector and Query&UI Service Added
an example for Agent Service

Closes: #472
Signed-off-by:Deepika Upadhyay <deepikaupadhyay01@gmail.com>

### CODE REVIEW

File: docs/deployment.md [hunk 1]
```diff
@@ -13,52 +13,39 @@ There are orchestration templates for running Jaeger with:
...
-## make sure to expose only the ports you use in your deployment scenario!
-docker run \
-  --rm \
-  -p5775:5775/udp \
-  -p6831:6831/udp \
-  -p6832:6832/udp \
-  -p5778:5778/tcp \
-  jaegertracing/jaeger-agent
+```sh
+MODE=test sh ./plugin/storage/cassandra/schema/create.sh | cqlsh
```
Comment (yurishkuro) [inline docs/deployment.md]: this should be done with a docker image instead

File: docs/deployment.md [hunk 2]
```diff
@@ -13,52 +13,39 @@ There are orchestration templates for running Jaeger with:
   * Kubernetes: [github.com/jaegertracing/jaeger-kubernetes](https://github.com/jaegertracing/jaeger-kubernetes),
   * OpenShift: [github.com/jaegertracing/jaeger-openshift](https://github.com/jaegertracing/jaeger-openshift).
 
-## Agent
+## Storage Backend
 
-Jaeger client libraries expect **jaeger-agent** process to run locally on each host.
-The agent exposes the following ports:
+Collectors require a persistent storage backend. Cassandra 3.x (default) and ElasticSearch are the
```
Comment (yurishkuro) [inline docs/deployment.md]: Cassandra 3.4+

File: docs/deployment.md [hunk 3]
```diff
@@ -13,52 +13,39 @@ There are orchestration templates for running Jaeger with:
...
+primary supported storage backends. There is ongoing work to add support for MySQL and ScyllaDB.
```
Comment (yurishkuro) [inline docs/deployment.md]: let's remove "There is ongoing work to add support for MySQL and ScyllaDB."

File: docs/deployment.md (hunk 1 above)
Comment (dexter816) [reply docs/deployment.md]: Can you tell the command , I am not able to find it exactly

Comment (jpkrohling) [reply docs/deployment.md]: https://github.com/jaegertracing/jaeger-kubernetes/blob/master/production/jaeger-production-template.yml#L108-L129

The image is just a wrapper around the script, so, it should be just a matter of running a command like this:

```
$ docker run -e MODE=... jaegertracing/jaeger-cassandra-schema
```

**Pull request closed without merging**